### ISSUE TITLE
Error when trying to merge into branch

### ISSUE THREAD

**Issue by honzik20** [initialcommit-com/git-sim#16]
Tried 'git-sim merge master':

```
(process:37120): GLib-GIO-WARNING **: 09:43:15.672: Unexpectedly, UWP app `Clipchamp.Clipchamp_2.5.15.0_neutral__yxz26nhyzhsrt' (AUMId `Clipchamp.Clipchamp_yxz26nhyzhsrt!App') supports 41 extensions but has no verbs
Traceback (most recent call last):
  File "C:\Python310\lib\runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Python310\lib\runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "C:\Python310\Scripts\git-sim.exe\__main__.py", line 7, in <module>
  File "C:\Python310\lib\site-packages\git_sim\__main__.py", line 89, in main
    scene.render()
  File "C:\Python310\lib\site-packages\manim\scene\scene.py", line 223, in render
    self.construct()
  File "C:\Python310\lib\site-packages\git_sim\git_sim.py", line 56, in construct
    self.command.execute()
  File "C:\Python310\lib\site-packages\git_sim\git_sim_merge.py", line 67, in execute
    self.draw_arrow_between_commits("abcdef", self.commits[0].hexsha)
  File "C:\Python310\lib\site-packages\git_sim\git_sim_base_command.py", line 486, in draw_arrow_between_commits
    end = self.drawnCommits[endsha].get_center()
KeyError: 'f54faa9550d5141eddac736b3f27214f8181aba0'
```

**Comment by initialcommit-io** [comment]
@honzik20 Thanks for reporting this.  Can you run the following command to help me understand the branching structure you have?  This will help me reproduce the issue:

```
$ git log --graph <branch1> <branch2>
```

where branch1 is your checked-out branch and branch2 is the one you are merging (master it looks like).  You can either copy the output of the command here or attach a screenshot.

**Comment by honzik20** [comment]
@initialcommit-io Sure thing:

```
PS S:\enterprise-Docs> git log --graph topics/jh/reorder-commission-docs master
* commit b99ef824fb6d7e0f8e7a0cf22e92a356bea6e69a (HEAD -> topics/jh/reorder-commission-docs, origin/topics/jh/reorder-commission-docs)
| Author: John Hayter <johnhayter@hotmail.com>
| Date:   Wed Jan 25 09:34:28 2023 +0000
|
|     Autodelete/autostart
|
* commit 1f1594f53824f8cb7591c6bf8a5f608f067c6484
| Author: John Hayter <johnhayter@hotmail.com>
| Date:   Wed Jan 25 09:34:07 2023 +0000
|
|     Update .gitignore
|
* commit afcc982f86a49821da4a2be5355e0d117c922aae
| Author: John Hayter <johnhayter@hotmail.com>
| Date:   Wed Jan 11 10:19:31 2023 +0000
|
|     WIP
|
*   commit 909910079c8c351b9864893450e1dbf8578e1cc2
|\  Merge: cbc2b7f 41bc610
| | Author: John Hayter <johnhayter@hotmail.com>
| | Date:   Mon Jan 9 14:30:17 2023 +0000
| |
| |     Merge branch 'master' into topics/jh/reorder-commission-docs
| |
* | commit cbc2b7fe57f7b75790c5afc4c7a96a230b7de597
| | Author: John Hayter <johnhayter@hotmail.com>
| | Date:   Wed Dec 28 16:41:55 2022 +0000
:...skipping...
* commit b99ef824fb6d7e0f8e7a0cf22e92a356bea6e69a (HEAD -> topics/jh/reorder-commission-docs, origin/topics/jh/reorder-commission-docs)
| Author: John Hayter <johnhayter@hotmail.com>
| Date:   Wed Jan 25 09:34:28 2023 +0000
|
|     Autodelete/autostart
|
* commit 1f1594f53824f8cb7591c6bf8a5f608f067c6484
| Author: John Hayter <johnhayter@hotmail.com>
| Date:   Wed Jan 25 09:34:07 2023 +0000
|
|     Update .gitignore
|
* commit afcc982f86a49821da4a2be5355e0d117c922aae
| Author: John Hayter <johnhayter@hotmail.com>
| Date:   Wed Jan 11 10:19:31 2023 +0000
|
|     WIP
|
*   commit 909910079c8c351b9864893450e1dbf8578e1cc2
|\  Merge: cbc2b7f 41bc610
| | Author: John Hayter <johnhayter@hotmail.com>
| | Date:   Mon Jan 9 14:30:17 2023 +0000
| |
| |     Merge branch 'master' into topics/jh/reorder-commission-docs
| |
* | commit cbc2b7fe57f7b75790c5afc4c7a96a230b7de597
| | Author: John Hayter <johnhayter@hotmail.com>
| | Date:   Wed Dec 28 16:41:55 2022 +0000
| |
| |     Solutions intro
| |
* | commit dffe360e7a14a4559c7d956ee3d3eeed7b5f2111
| | Author: John Hayter <johnhayter@hotmail.com>
| | Date:   Wed Dec 28 16:39:56 2022 +0000
| |
| |     SEMS
| |
* | commit 16ba748a099b89b05675c0b3afc9fd3ddbdc536e
| | Author: John Hayter <johnhayter@hotmail.com>
| | Date:   Wed Dec 28 16:39:50 2022 +0000
| |
| |     Gas bottles
| |
* | commit 899d64d132ca42e6463a47a2c1fd9056bd61615e
| | Author: John Hayter <johnhayter@hotmail.com>
| | Date:   Fri Dec 23 09:12:32 2022 +0000
| |
| |     WIP
| |
* |   commit 08ed25d3761040621bff94850ae89bc75080d943
|\ \  Merge: 6b6fae3 530116c
| | | Author: John Hayter <johnhayter@hotmail.com>
| | | Date:   Wed Dec 21 16:11:02 2022 +0000
:
```

**Comment by initialcommit-io** [comment]
@honzik20 I just pushed a fix to this issue in git-sim version 0.1.5.

Please upgrade using `pip install git-sim --upgrade` and let me know if any issues after you retest.

**Comment by honzik20** [comment]
Looks good - thanks @initialcommit-io. Tool looks promising!
